[2022-03-17 03:20:53,599] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest 2019-06-02T06:00:00+00:00 [queued]>
[2022-03-17 03:20:53,637] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest 2019-06-02T06:00:00+00:00 [queued]>
[2022-03-17 03:20:53,639] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 03:20:53,640] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-03-17 03:20:53,642] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 03:20:53,666] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): ingest> on 2019-06-02T06:00:00+00:00
[2022-03-17 03:20:53,671] {standard_task_runner.py:52} INFO - Started process 895 to run task
[2022-03-17 03:20:53,685] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'testing_dag', 'ingest', '2019-06-02T06:00:00+00:00', '--job-id', '17', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/testing_dag.py', '--cfg-path', '/tmp/tmpqypnvoyi', '--error-file', '/tmp/tmpb4gcmpab']
[2022-03-17 03:20:53,691] {standard_task_runner.py:77} INFO - Job 17: Subtask ingest
[2022-03-17 03:20:53,787] {logging_mixin.py:104} INFO - Running <TaskInstance: testing_dag.ingest 2019-06-02T06:00:00+00:00 [running]> on host 509d5aa4e827
[2022-03-17 03:20:53,897] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=testing_dag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2019-06-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-06-02T06:00:00+00:00
[2022-03-17 03:20:53,899] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-03-17 03:20:53,904] {bash.py:158} INFO - Running command: ls /opt/airflow
[2022-03-17 03:20:53,922] {bash.py:169} INFO - Output:
[2022-03-17 03:20:53,999] {bash.py:173} INFO - airflow.cfg
[2022-03-17 03:20:54,002] {bash.py:173} INFO - dags
[2022-03-17 03:20:54,004] {bash.py:173} INFO - google_credentials.json
[2022-03-17 03:20:54,005] {bash.py:173} INFO - logs
[2022-03-17 03:20:54,008] {bash.py:173} INFO - output_2019-01.csv
[2022-03-17 03:20:54,009] {bash.py:173} INFO - output_2019-02.csv
[2022-03-17 03:20:54,010] {bash.py:173} INFO - output_2019-03.csv
[2022-03-17 03:20:54,012] {bash.py:173} INFO - output_2019-04.csv
[2022-03-17 03:20:54,013] {bash.py:173} INFO - output_2019-05.csv
[2022-03-17 03:20:54,014] {bash.py:173} INFO - output_2019-06.csv
[2022-03-17 03:20:54,016] {bash.py:173} INFO - plugins
[2022-03-17 03:20:54,017] {bash.py:173} INFO - unittests.cfg
[2022-03-17 03:20:54,018] {bash.py:173} INFO - webserver_config.py
[2022-03-17 03:20:54,020] {bash.py:177} INFO - Command exited with return code 0
[2022-03-17 03:20:54,072] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=testing_dag, task_id=ingest, execution_date=20190602T060000, start_date=20220317T032053, end_date=20220317T032054
[2022-03-17 03:20:54,129] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-17 03:20:54,170] {local_task_job.py:146} INFO - Task exited with return code 0
